config.json

```diff
@@ -40,7 +40,7 @@
       "description": "MsBuild logging options.",
       "valueType": "passThrough",
       "values": [],
-      "defaultValue": "/l:BinClashLogger,Tools/net46/Microsoft.DotNet.Build.Tasks.dll;LogFile=binclash.log"
```

Comment:
What problem did this cause in the Linux build?  What is the impact on the Windows build by removing these settings?